### DISCUSSION TITLE
Message regarding AR availability on desktop browsers

### DISCUSSION THREAD

**Discussion by devhims** [google/model-viewer#2025] [Ideas]
We only get the 3d view on desktop browsers. I would like to leave a message informing the users that 'AR is unavailable on the current platform'. Is there any property I can check in the model viewer component?

**Comment by elalish** [comment]
You can, though it won't be perfect as even we sometimes show the AR button when AR isn't really available (it's heuristic). You could either use similar heuristics as we use (checking navigator string). Actually it seems like the right thing for me to do is add another ARStatus value: not-supported. Then you could activate this based on CSS.
